ci/cron/src/Main.hs

```diff
@@ -271,18 +272,25 @@ docs = do
 
 download_assets :: FilePath -> GitHubRelease -> IO ()
 download_assets tmp release = do
+    manager <- HTTP.newManager TLS.tlsManagerSettings
     tokens <- Control.Concurrent.QSem.newQSem 20
     Control.Concurrent.Async.forConcurrently_ (map uri $ assets release) (\url ->
         Control.Exception.bracket_
           (Control.Concurrent.QSem.waitQSem tokens)
           (Control.Concurrent.QSem.signalQSem tokens)
           (do
-        shell_ $ unlines ["bash -c '",
-            "set -euo pipefail",
-            "eval \"$(dev-env/bin/dade assist)\"",
-            "cd \"" <> tmp <> "\"",
-            "wget --quiet \"" <> show url <> "\"",
-            "'"]))
+              req' <- HTTP.parseRequest (show url)
+              let req = req' { HTTP.requestHeaders = [("User-Agent", "DAML cron (team-daml-language@digitalasset.com)")] }
+              HTTP.withResponse req manager (\resp -> do
+                  let body = HTTP.responseBody resp
```

Comment:
You are basically reimplementing what `http-conduit` gives you. With that, you would end up with something like
```
runResourceT $ HTTP.responseBody resp  $$ sinkFile (tmp </> …)`
```

I don’t particularly mind that so feel free to ignore that suggestion or to address it separately.

Reply:
I did come upon references to `conduit`, but it seemed to add a lot of concepts I do not know yet. I imagine it's not a lot of complexity once you know them, but this is also not very complex as it stands. I tend to be a bit reluctant to add dependencies for a single function. I'll take a closer look at conduit if it comes up again.